### DISCUSSION TITLE
Custom Links with boolean conditional

### DISCUSSION THREAD

**Discussion by JoeOster** [netbox-community/netbox#14063] [Help Wanted!] [answered]
I am trying to create a "Custom Links" links option for Netbox. I have used conditional formatting for other links and had no issues, but the Boolean isn't working. 

Scenario: Create a button, if the custom field "web UI" is equal to true then populate the link with the IPv4 address

- Text: {% if object.cf['webUI']==true%}Web UI{% endif %}
- URL:  https://{{ object.primary_ip4.address.ip}}

**Comment by candlerb** [comment]
"Isn't working" is an unclear description of the problem. What do you actually see? Do you get an exception pop-up? Does the Web UI button not appear at all, or is greyed out? Does it appear, but not work?

And what Netbox version are you using?

**Reply by JoeOster** [reply]
We are using Net box 3.6.2

I am saying that the filter isn't working within "Link Text" despite having a check applied to the custom field [webUI] option within Net box, yet the link does not populate.  Below is from the API

`
 {
           "custom_fields": {
                "WebUI": true
},},
`

Either we can't filter on boolean values or I just couldnt craft the filter correctly

`
Text: {% if object.cf['webUI']==true%}Web UI{% endif %}
`

**Reply by candlerb** [reply]
JSON is case-sensitive. `WebUI` is not the same as `webUI`

**Reply by JoeOster** [reply, marked as answer]
Good catch, I was crafting it correctly. I think the problem was I tried so many things, that I modified the cf and didn't track back to the link...
`
